RiskChanges/RiskChangesOps/readvector.py

```diff
@@ -1,12 +1,11 @@
 import geopandas as gpd
-import RiskChangesOps.readmeta as readmeta
-import RiskChangesOps.readvulnerability as readvulnerability
+from . import readmeta
 import psycopg2
 import pandas as pd 
 def readear(connstr,earid):
     engine=psycopg2.connect(connstr)
     metatable=readmeta.earmeta(connstr,earid)
-    eartablename=metatable.table_name[0]
+    eartablename=metatable.layer_name[0]
```

Comment:
Actually, I used the layer_name field that's why I replaced it here.

Reply:
but in the database i had it was table_name isn't it? did the database change? it is fine if it works for you.

Reply:
Yes. But when I checked I found that I used `layer_name`. the same field is used in geoserver as well. so

Reply:
okay great. if it works it should be okay  👯‍♂️ 🍡

Reply:
Ya sure. Thank you for your confirmation.